core/joypad/joypad.ts

```diff
@@ -204,7 +203,7 @@ function _pressJoypadButton(buttonId: i32): void {
     }
 
     // Determine if we should request an interrupt
-    let joypadRegister: i32 = Joypad.joypadRegisterFlipped;
+    // let joypadRegister: i32 = Joypad.joypadRegisterFlipped; // not used
```

Comment:
Oh yeah, this line can be remove for not being used, but the one above comment is still accurate 😄